Scripts/post_build.ps1

```diff
@@ -36,7 +36,7 @@ function Copy-Resources ($path, $config) {
     $target = "$output\$config"
     Copy-Item -Recurse -Force $project\Themes\* $target\Themes\
     Copy-Item -Recurse -Force $project\Images\* $target\Images\
-    Copy-Item -Recurse -Force $path\Plugins\HelloWorldPython $target\Plugins\HelloWorldPython
+    Copy-Item -Recurse -Force $path\Plugins\HelloWorldPython\* $target\Plugins\HelloWorldPython
```

Comment:
My output of HelloWorldPython **was** wrong. I had a extra HelloWorldPython in the path. My output was

`Output\Debug\Plugins\HelloWorldPython\HelloWorldPython\*`

Currently I can't reproduce it. Please skip the change